frontend/pages/project-developers/edit.tsx

```diff
@@ -0,0 +1,191 @@
+import { useState, useCallback, useEffect } from 'react';
+
+import { Path, SubmitHandler, useForm } from 'react-hook-form';
+import { useIntl } from 'react-intl';
+import { dehydrate, QueryClient } from 'react-query';
+
+import { useRouter } from 'next/router';
+
+import { entries } from 'lodash-es';
+import { InferGetStaticPropsType } from 'next';
+
+import useInterests from 'hooks/useInterests';
+
+import { loadI18nMessages } from 'helpers/i18n';
+import { getProjectDeveloperFormItems, getServiceErrors } from 'helpers/pages';
+
+import LeaveFormModal from 'containers/leave-form-modal';
+import MultiPageLayout, { Page } from 'containers/multi-page-layout';
+import { About, Profile } from 'containers/project-developer-form-pages';
+
+import Head from 'components/head';
+import { Paths, Queries, UserRoles } from 'enums';
+import FormPageLayout, { FormPageLayoutProps } from 'layouts/form-page';
+import ProtectedPage from 'layouts/protected-page';
+import { PageComponent } from 'types';
+import { ProjectDeveloperSetupForm } from 'types/projectDeveloper';
+import useProjectDeveloperValidation, { formPageInputs } from 'validations/projectDeveloper';
+
+import { useUpdateProjectDeveloper } from 'services/account';
+import { getEnums, useEnums } from 'services/enums/enumService';
+import { useCurrentProjectDeveloper } from 'services/project-developers/projectDevelopersService';
+
+export async function getStaticProps(ctx) {
+  const queryClient = new QueryClient();
+  queryClient.prefetchQuery(Queries.EnumList, getEnums);
+  return {
+    props: {
+      intlMessages: await loadI18nMessages(ctx),
+      dehydratedState: dehydrate(queryClient),
+    },
+  };
+}
+
+const LAST_PAGE = 1;
```

Comment:
I don't think we should hardcode this (even though it's technically also hardcoded in the other form). 

The `MultiPageLayout` component knows [how many pages](https://github.com/Vizzuality/heco-invest/blob/develop/frontend/containers/multi-page-layout/component.tsx#L42) there are in total; exlucing the `Outro` page (which I'm not sure is being used). 

Not the most elegant of solutions, but a callback could be used when it loads to tell you how many pages there are in total (and then you'd know which one is the last). 

I feel like this hardcoded number is easy to miss, and because the form is duplicated (new vs create), if we add another page in the future there's a big chance we'll forget to update this in both forms.